### PR TITLE
MAGECLOUD-5181: Disable cron container by default

### PR DESCRIPTION
**Pull request by shiftedreality**
### Fixed Issues (if relevant)

1. https://github.com/magento/magento-cloud-docker/issues/139
2. https://magento2.atlassian.net/browse/MAGECLOUD-5181

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages

### CODE REVIEW

File: src/Command/BuildCompose.php [hunk 1]
```diff
@@ -210,10 +210,16 @@ protected function configure(): void
                 DeveloperBuilder::SYNC_ENGINE_NATIVE
             )
             ->addOption(
-                self::OPTION_NO_CRON,
+                'no-cron',
```
Comment (bbatsche) [inline src/Command/BuildCompose.php:213]: Why not leave this constant?

**Review by bbatsche — APPROVED** [review]
I think it's fine, just want some clarity on one question

**Comment by NadiyaS** [comment]
@YPyltiai Do you approve this BIC?
and there are some BIC in code itself

**Comment by YPyltiai** [comment]
> @YPyltiai Do you approve this BIC?
> and there are some BIC in code itself

My main concern is that cron parameter is releasing in 1.0.0 tomorrow and we already change it. Maybe we get this into the release to not introduce BIC day after we drop the release? I am okay with the change itself but don't like the timing and feel it should be in current release to avoid confusion.

CC: @shiftedreality

**Pull request closed without merging**